README.md

```diff
@@ -1,2 +1,251 @@
+    </ul>
+  </li>
+  <li>Webpack 5</li>
+  <li>Follow Microverse's list of best practices for JavaScript projects.</li>
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- 🎤 [Project presentation: video demo](https://drive.google.com/file/d/1-eSDabquIJCbvbQblqBPvSwbDIvaRfIb/view?usp=sharing)
+
+- [Live Demo Link](https://iturres.github.io/TVmaze-movies-website/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li><p>A modern browser (supports html5)</p></li>
+</ul>
+<ul>
+    <li><p>Git and a GitHub account</p></li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder by using this 👇️ command :
+
+```
+git clone https://github.com/ITurres/TVmaze-movies-website.git
+```
+
+### Install
+
+Install this project with:
+
+```
+ npm install
+```
+
+### Usage
+
+```
+ npm run start
+```
+
+### Run tests
+
+```
+ npm test
+```
+
+### Deployment
+
+- Not aplicable.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Arthur Iturres**
+
+- GitHub: [@ITurres](https://github.com/ITurres)
+- Twitter: [@Arthur_ITurres](https://twitter.com/ArthurIturres)
+- LinkedIn: [Arthur Emanuel G. Iturres](https://www.linkedin.com/in/arturoemanuelguerraiturres/)
+
+👤 **Ruth Abreu**
+
+- GitHub: [@Ruthmy](https://github.com/Ruthmy)
+- Twitter: [@rury_exe](https://twitter.com/rury_exe)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ruth-abreu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Upcoming improvements:
+
+- [x] All fulfilled.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project and know someone who might find it helpful, please share it.
+Or give it a **star** ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I thank Microverse for this fantastic opportunity, the code reviewers for their advice and time, and my excellent coding partner and team member [@Ruthmy](https://github.com/Ruthmy) 🏆
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- Hey team, the MIT link is currently broken, please, kindly fix this to make your readme more professional.

Reply:
Hi @zieeco ! How are you? thank you! yes, you are absolutely right, the link is currently not pointing to where it should. We will make make the changes right away 🎯